### DISCUSSION TITLE
How do you display non-code (i.e. rendered HTML) sections within Monaco panes?

### DISCUSSION THREAD

**Discussion by tgross35** [microsoft/monaco-editor#3861] [Q&A] [answered]
Is there a way using the library to render HTML within one of the resizable/tabbed panes, without having the code editor?

I'm looking to be able to accomplish something like the welcome or settings screens, which act like normal panes but don't do anything with code.

![image](https://user-images.githubusercontent.com/13724985/227393917-6192ed7d-1ae7-45cf-b89b-ac4a915a410e.png)

![image](https://user-images.githubusercontent.com/13724985/227394017-f47931d5-2805-4886-9cc2-1a3f28067cbb.png)

**Comment by tamayika** [comment, marked as answer]
This question is not related to monaco-editor, please search on stackoverflow or other sites for a way to achieve a tabbed UI.

**Reply by tgross35** [reply]
Hm I was under the impression that panes support was part of the editor component (because of diff), but I guess that's incorrect. Thanks for the followup.
